src/styles/blocks/cta.scss

```diff
@@ -0,0 +1,91 @@
+.cta {
+  background-color: #fff;
+  width: 100vw;
+  height: 571px;
+  margin: 120px 0;
+
+  @include on-tablet {
+    height: 796px;
+    margin: 200px 0 -75px;
+    background-image: url(../images/Shapes.png);
+    object-fit: cover;
+    object-position: 25% 50%;
+  }
+
+  @include on-small-desktop {
+    margin: 186px 0 -142px;
+    height: 888px;
+  }
+
+  &__text {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding: 72px 20px;
+    text-align: center;
+
+    @include on-tablet {
+      width: 100vw;
+      margin: 0;
+      padding: 200px 0 0;
+    }
+
+    &-title {
+      font-size: 44px;
+      font-weight: 600;
+      margin: 0 0 24px;
+      color: #253757;
+      letter-spacing: -1px;
+
+      @include on-tablet {
+        font-size: 52px;
+        width: 400px;
+      }
+
+      @include on-small-desktop {
+        width: 600px;
+      }
+    }
+
+    &-content {
+      font-size: 23px;
+      font-weight: 600;
+      color: #6C788B;
+      font-family: Open-Sans, sans-serif;
+
+      @include on-tablet {
+        display: flex;
+        flex-direction: column;
+        padding: 0 66px;
+      }
+
+      @include on-small-desktop {
+        padding: 0 200px;
+      }
+
+      @include on-desktop {
+        padding: 0 320px;
+      }
+    }
+
+    .btn {
```

Comment:
Avoid using relative position between elements to apply styles; this leads to highly coupled code; which is harder to maintain and less reusable